src/utils.js

```diff
@@ -616,6 +616,69 @@ export const snakeCaseToCamelCase = (snakeCaseStr) => {
         .join('');
 };
 
+/**
+ * Scrolls to the bottom of a page, or until it times out.
+ * @param {Object} page
```

Comment:
What is the page, a URL, HTML, or perhaps Puppeteer object? I don't know, the docs don't say